README.md

```diff
@@ -312,6 +312,37 @@ class CustomSlide extends FlutterDeckSlideWidget {
 }
 ```
 
+### BigFact slide
+
+To create a title slide, use the `FlutterDeckSlide.bigFact` constructor. It is responsible for rendering Big Title in the center with description below it if needed also is uses the default header and footer configuration
+```dart
+class BigFactSlide extends FlutterDeckSlideWidget {
```

Comment:
Do not forget to update this code snippet after making changes in the `big_fact_slide.dart`.